charmcraft.yaml

```diff
@@ -94,6 +94,14 @@ requires:
     optional: true
     limit: 1
     description: Forward telemetry to another Observability stack (Grafana Cloud, COS, etc.).
+  certificates:
```

Comment:
Should we call this relation `server-cert`?
In any case, it's not certificate**s** in plural.